fix(bil): docs update (#4485)
scaleway/docs-content: pages/billing/troubleshooting/cannot-add-payment-method.mdx

---
meta:
  title: I cannot add a payment method

>   description: Resolve issues encountered when adding a new payment method.

Reviewer: ```suggestion
  description: Resolve issues encountered when adding a new payment method to your Scaleway account.
```

content:
  h1: I cannot add a payment method

>   paragraph: Resolve issues encountered when adding a new payment method.

Reviewer: ```suggestion
  paragraph: Resolve issues encountered when adding a new payment method to your Scaleway account.
```

tags: billing credit card 
dates:
  validation: 2025-02-24
  posted: 2025-02-24
categories:
  - billing
---

## I can't find my credit card validation code

### Cause
When adding a new credit card to your Scaleway account, a €1 authorization charge is made to verify the card's validity. This transaction includes a 4-digit validation code which appears on your bank statement. It typically appears on your bank statement within 24 hours. 

### Solution

> - **Wait for the code to appear**: the validation code typically appears on your bank statement within 24 hours. If you don't see it immediately, please wait for this period.

Reviewer: ```suggestion
- **Wait for the code to appear**: the validation code typically appears on your bank statement within 24 hours. If you do not see it immediately, please wait for this period.
```
